tools/_setup_generation/step_refman.py

```diff
@@ -382,8 +382,8 @@ def generate_entries(
                             name,
                             TYPE_ID,
                             package,
-                            entry_info["module"],
-                            entry_info.get("packages"),
+                            type["module"],
```

Comment:
why not type.get("module") ?

Reply:
Any significant difference at this point?

Reply:
For consistency sake
Get() would never throw